setup/gcp/install_setup.py

```diff
@@ -54,9 +58,19 @@ def run():
                               help='Notification recipient email')
     email_params.add_argument('--gsuite-superadmin-email',
                               help='G Suite super admin email')
-
     args = vars(parser.parse_args())
-    forseti_setup = gcloud_env.ForsetiGcpSetup(**args)
+
+    if not args.get('type'):
```

Comment:
I don't think we should do this implicitly when there is no type.  And why should we support both CLI/Sever on the same VM?  If we really have to do this, it should be an explicit option that user has to pick.  And if no type is given, then installation will abort.

Reply:
Just to clarify, when nothing is specified, we will install both server and CLI in 2 different VMs. The server VM will always have it's own CLI running as a cron job.

Reply:
+ @mirons-google

Reply:
I see, this is to deploy to different VM.  Would still be nice to have something explicit, like `server_and_client`.  But I will defer that what the team thinks would work best.